src/components/box/index.stories.mdx

```diff
@@ -2,7 +2,7 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
 
 import Box from './index.tsx';
 
-<Meta title="Patterns/Box" component={Box} />
+<Meta title="Components/Utils/Box" component={Box} />
```

Comment:
Consider moving the box to the layout

Reply:
Yes, it makes more sense in the layout category. Thanks!